addons/loyalty/views/loyalty_card_views.xml

```diff
@@ -11,13 +11,30 @@
                             <field name="code" readonly="1"/>
                             <field name="expiration_date"/>
                             <field name="partner_id"/>
-                            <label string="Balance" for="points"/>
-                            <span class="d-inline-block">
-                                <field name="points" class="w-auto oe_inline me-1"/>
-                                <field name="point_name" no_label="1" class="d-inline"/>
-                            </span>
+                            <label string="Balance" for="points_display"/>
+                            <button
+                                name="%(loyalty_update_balance_wizard_action)d"
```

Comment:
You should create object method in python to pass default values like old balance
or context to in action method to set default values in update balance wizard